activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -426,14 +426,22 @@ def columns(table_name) # :nodoc:
           each_hash(result).map do |field|
             type_metadata = fetch_type_metadata(field[:Type], field[:Extra])
             if type_metadata.type == :datetime && field[:Default] == "CURRENT_TIMESTAMP"
-              new_column(field[:Field], nil, type_metadata, field[:Null] == "YES", field[:Default], field[:Collation])
+              new_column(field[:Field], nil, type_metadata, field[:Null] == "YES", field[:Default], field[:Collation], field[:Comment].presence)
             else
-              new_column(field[:Field], field[:Default], type_metadata, field[:Null] == "YES", nil, field[:Collation])
+              new_column(field[:Field], field[:Default], type_metadata, field[:Null] == "YES", nil, field[:Collation], field[:Comment].presence)
             end
           end
         end
       end
 
+      def table_comment(table_name)
+        select_value(<<-MySQL.strip_heredoc, 'SCHEMA')
+          SELECT table_comment
+          FROM INFORMATION_SCHEMA.TABLES
+          WHERE table_name=#{quote(table_name)};
+        MySQL
```

Comment:
Some new editors (like Atom or RubyMine) displays SQL dialect-specific syntax highlighting with such a heredocs. But since I'm not using MySQL-specific things here, I changed back to `SQL`.